tests/Unit/Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/Test_Wedge.cpp

```diff
@@ -204,54 +222,182 @@ void test_only_transition() {
   set_orig_rad_over_rad(std::array{0.0, 0.0, 0.2 * inner_radius}, 0.4);
   CHECK(orig_rad_over_rad.has_value());
   CHECK(orig_rad_over_rad.value() == approx(5.0));
+}
+
+void test_only_transition_offset() {
+  INFO("Test only transition offset");
+  const double inner_sphericity = 1.0;
+  const double outer_sphericity = 0.0;
+  const std::array<double, 3> inner_center{-1.0, -1.0, 0.0};
+  const std::array<double, 3> outer_center{0.0, 0.0, 0.0};
+
+  const Wedge wedge =
+      make_wedge(inner_sphericity, outer_sphericity, inner_center, outer_center,
+                 Wedge::Axis::PlusX);
+
+  // All these values were calculated by hand given the centers above and this
+  // specific point below
+  const double half_cube_length = outer_radius / sqrt(3.0);
+  const std::array<double, 3> point{4.0, 0.0, 0.0};
+  const std::array<double, 3> centered_point = point - inner_center;
+  const double radius_from_inner_center = magnitude(centered_point);
+
+  const double inner_distance = 0.5;
+  const double outer_distance = (half_cube_length + 1) * sqrt(26.0) / 5.0;
+  const double distance_difference = outer_distance - inner_distance;
+
+  const double function_value =
+      (outer_distance - radius_from_inner_center) / distance_difference;
+
+  CHECK(wedge(centered_point) == approx(function_value));
+  const std::array<double, 3> projection_center = inner_center - outer_center;
+  // Because of PlusX
+  const std::array<double, 3> outer_gradient =
+      centered_point / radius_from_inner_center *
+          (half_cube_length - projection_center[0]) / centered_point[0] -
+      std::array{
+          (half_cube_length - projection_center[0]) / square(centered_point[0]),
+          0.0, 0.0} *
+          radius_from_inner_center;
+  const std::array<double, 3> expected_gradient =
+      (outer_gradient - centered_point / radius_from_inner_center) /
+          distance_difference -
+      ((outer_distance - radius_from_inner_center) * outer_gradient /
+       square(distance_difference));
+  CHECK_ITERABLE_APPROX(wedge.gradient(centered_point), expected_gradient);
+
+  std::optional<double> orig_rad_over_rad{};
+  const auto set_orig_rad_over_rad =
+      [&](const std::array<double, 3>& mapped_point,
+          const double distorted_radii) {
+        orig_rad_over_rad =
+            wedge.original_radius_over_radius(mapped_point, distorted_radii);
+      };
+
+  // Test actual values
+  set_orig_rad_over_rad(centered_point, 0.0);
+  CHECK(orig_rad_over_rad.has_value());
+  CHECK(orig_rad_over_rad.value() == approx(1.0));
+
+  std::array<double, 3> mapped_point =
+      centered_point * (1.0 - function_value / radius_from_inner_center * 0.5);
+  set_orig_rad_over_rad(mapped_point, 0.5);
+  CHECK(orig_rad_over_rad.has_value());
+  CHECK(orig_rad_over_rad.value() ==
```

Comment:
You can do `value_or(-std::limits<double>::max())`. That would do both checks and maybe make clang-tidy happy?